### ISSUE TITLE
Issue: Cannot specify another admin user to run a command from the gsudo options

### ISSUE THREAD

**Issue by sebma** [gerardog/gsudo#150]
## Issue Description

My `$env:username` AD account is not admin of my workstation but I've been created another user for all the admin tasks.
How can I specify another admin user to run the command from the CLI  : like Linux `sudo -u $user ...` ?

## Steps to Reproduce

```powershell
PS C:\Users> gsudo status | findstr -i Admin:
  Is Admin: False
PS C:\Users> gsudo -u t2-xyz@$env:USERDNSDOMAIN dir
Error: Invalid option: -u
```

## Context:
 - Windows version: `Windows 10 Pro 21H2`
 - Powershell version : `5.1.19041.1682` (taken from `$PSVersionTable.PSVersion.ToString()`)
 - gsudo version:  `gsudo v1.3.0 (Branch.master.Sha.24fb735f547e1e5dd7aa22fdd77777fa8c923a1c)`

**Comment by gerardog** [comment]
Implemented in #188

**Comment by gerardog** [comment]
Pre-Released: https://github.com/gerardog/gsudo/releases/tag/v2.0.0-preview
